### DISCUSSION TITLE
I'd like to be able to wrap fields with long values (e.g., title field, label field) to multiple rows in table view.

### DISCUSSION THREAD

**Discussion by michaellenaghan** [community/community#7430] [Projects and Issues]
I tag issues with multiple labels. In table view, using the default column width, I can only see one and a half lables per issue. I can make the column wider, but that of course just forces me to scroll horizontally to see the fields that got pushed off.

I find the "not wrapping" thing to also be an issue for item titles. I've actually edited titles to make them short enough to fit. That doesn't seem ideal.

(The board view works well; the full title and all labels are shown.)

**Comment by benjamin-popp-zap-llc** [comment]
Aside from **Titles** and **Labels**, wrapping would also make sense for **Assignees**. If an issue has multiple assignees, wrapping to multiple lines would be very helpful.

I imagine this as an option in the table configuration:

<img width="307" height="161" alt="image" src="https://github.com/user-attachments/assets/435fa69c-e03a-4ffc-a9fb-8a41c4eefa04" />

`Wide Content:              Truncate / Wrap >`
